tests/python/unittest/test_meta_schedule_schedule_rule_multi_level_tiling.py

```diff
@@ -883,13 +883,13 @@ def test_cuda_tensor_core_matmul_relu_global():
 sch.compute_at(block=b70, loop=l46, preserve_unit_loops=True)
 l71, l72, l73, l74, l75, l76 = sch.get_loops(block=b70)
 l77 = sch.fuse(l75, l76, preserve_unit_iters=True)
-v78 = sch.sample_categorical(candidates=[1, 2, 3, 4], probs=[0.25, 0.25, 0.25, 0.25])
+v78 = sch.sample_categorical(candidates=[1, 2, 4, 8], probs=[0.25, 0.25, 0.25, 0.25])
 sch.annotate(block_or_loop=b70, ann_key="meta_schedule.cooperative_fetch", ann_val=v78)
 b79 = sch.cache_read(block=b19, read_buffer_index=1, storage_scope="shared")
 sch.compute_at(block=b79, loop=l46, preserve_unit_loops=True)
 l80, l81, l82, l83, l84, l85 = sch.get_loops(block=b79)
 l86 = sch.fuse(l84, l85, preserve_unit_iters=True)
-v87 = sch.sample_categorical(candidates=[1, 2, 3, 4], probs=[0.25, 0.25, 0.25, 0.25])
+v87 = sch.sample_categorical(candidates=[1, 2, 4, 8], probs=[0.25, 0.25, 0.25, 0.25])
```

Comment:
BTW, i'm not sure if we should rule out `float3` in CUDA

Reply:
This is fp16

Reply:
That makes sense!